src/resources/2020/RetroCape.ts

```diff
@@ -0,0 +1,124 @@
+import { cliExecute, Skill } from "kolmafia";
+import { have as haveItem } from "../../lib";
+import { Modifiers } from "../../modifier";
+import { get } from "../../property";
+import { $item, $skill } from "../../template-string";
+
+export const item = $item`unwrapped knock-off retro superhero cape`;
+/**
+ * Determines whether you have() the Retro Cape.
+ * @returns Whether you have the Retro Cape available.
+ */
+export function have(): boolean {
+  return haveItem(item);
+}
+
+const Heroes = {
+  vampire: {
+    "Muscle Percent": 30,
+    "Maximum HP": 50,
+  },
+  heck: {
+    "Mysticality Percent": 30,
+    "Maximum MP": 50,
+  },
+  robot: {
+    "Moxie Percent": 30,
+    "Maximum HP": 25,
+    "Maximum MP": 25,
+  },
+};
+type Hero = keyof typeof Heroes;
+type Mode = "hold" | "thrill" | "kiss" | "kill";
+
+export const currentHero = () => get("retroCapeSuperhero") as Hero;
+export const currentMode = () => get("retroCapeWashingInstructions") as Mode;
+
+const modeMap = new Map<[Hero, Mode], Modifiers>([
+  [
+    ["vampire", "hold"],
```

Comment:
Did you consider structing it as:
```
{
  vampire: {
    hold: {},
    thrill: {}
}
```
It's not a huge deal since it isn't exported, but your setup struck me as a bit confusing at first.
I guess it can't be a map that way, but a nested object makes more sense to me for this data.